MaxText/pyconfig.py

```diff
@@ -85,12 +85,19 @@ def validate_profiler_type(s: str) -> None:
     raise ValueError("Invalid profiler type was passed. Valid options ", valid_profiler_types)
 
 
+def validate_model_call_mode(s: str) -> None:
+  valid_model_call_modes = ("", "inference")
+  if s not in valid_model_call_modes:  # currently supported attention
+    raise ValueError("Invalid model call mode. Valid options ", valid_model_call_modes)
```

Comment:
Nit:
raise ValueError(f"Invalid model call mode {s}. Valid options {valid_model_call_modes}")

It will help in debugging.